AIDevGallery/Telemetry/Events/SampleInteractionEvent.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using AIDevGallery.Models;
+using Microsoft.Diagnostics.Telemetry;
+using Microsoft.Diagnostics.Telemetry.Internal;
+using System;
+using System.Diagnostics.Tracing;
+
+namespace AIDevGallery.Telemetry.Events;
+
+[EventData]
+internal class SampleInteractionEvent : EventBase
+{
+    public override PartA_PrivTags PartA_PrivTags => PrivTags.ProductAndServiceUsage;
+
+    public string SampleId { get; }
+    public string Model1Id { get; }
```

Comment:
why not make a models array instead of hardcoding model1, model2, etc?

Reply:
Also, since these are not just models - we should think about how we make this more readable and move away from just calling everything model

Reply:
These are reflected by the telemetry [EventData], so we should be very clear with what we call them.
We do not have plans on having samples with more than two models/endpoints (only the rag and multipose samples uses this). Adding more than two would make the UI very complex too.